package.json

```diff
@@ -8,6 +8,7 @@
     "server": "node --harmony_destructuring --harmony_rest_parameters server/index.js",
     "client": "webpack-dev-server --inline --watch",
     "test": "karma start",
+    "tdd": "karma start --no-single-run --auto-watch --reporters nyan",
```

Comment:
usually its better to add such improvements as a separate PR: it makes it easier to manage git.
at least they can be faster merged to common branch